xml/System/Convert.xml

```diff
@@ -18554,11 +18557,12 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="2" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="chars">To be added.</param>
-        <param name="bytes">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="chars">A span containing the string representation that is encoded with base-64 digits.</param>
+        <param name="bytes">When this method returns <see langword="true" />, the converted 8-bit unsigned integers.</param>
```

Comment:
@mairaw what do you think about this suggestion?:

```suggestion
        <param name="bytes">When this method returns <see langword="true" />, the converted 8-bit unsigned integers. When this method returns <see langword="false" />, either the span remains unmodified (length zero) or contains an incomplete conversion of <paramref name="chars" />, up to the last valid character.</param>
```

Reply:
Depending on your answer, I'll apply a similar suggestion to the rest of the places where you asked me about a false case.

Reply:
Looks good @carlossanlop!